### ISSUE TITLE
Trying to convert bezierlanenet to onnx

### ISSUE THREAD

**Issue by farouk09** [voldemortX/pytorch-auto-drive#97]
Hi,

I'm trying to convert bezierlanenet to onnx, first i got this error:

`RuntimeError: Exporting the operator flip to ONNX opset version 9 is not supported. Support for this operator was added in version 10, try exporting with this version.`

I have changed the oppset version to 10 but now i'm getting this error, any helps !

`RuntimeError: ArrayRef: invalid index Index = 2; Length = 2`

**Comment by voldemortX** [comment]
I don't think it is supported yet (and I remember saying that in the documentation?), considering the deformable conv from mmcv. Although technically mmcv supports it in beta versions.

**Comment by voldemortX** [comment]
Perhaps we can do something like in here:
https://github.com/shouxieai/tensorRT_Pro/tree/main/tutorial/2.0CenterNet_from_torch_trt
